src/utils/gamehelper.js

```diff
@@ -236,6 +270,16 @@ class GameHelper{
         return {count :count,symbol : symbol};
     }
 
+    /**
+     * this function use random Wild multipair 
+     * @returns wild multipair
+     */
+    randomWildMul(){
+        let wildMult = [2,4,6]
+        const element = Math.floor(Math.random() * ((2-0)+ 1) + 0);
```

Comment:
` Math.floor(Math.random() * ((2-0)+ 1) + 0);`
it can be written as `Math.floor(Math.random() * wildMult.length());`